tools/cell_census_builder/__main__.py

```diff
@@ -72,7 +72,6 @@ def main() -> int:
     process_initializer(args.verbose)
 
     # normalize our base URI - must include trailing slash
-    args.uri = args.uri if args.uri.endswith("/") else args.uri + "/"
```

Comment:
explain why this is safe? I'm not sure that it's _not_, just want to understand.

Reply:
we are doing it [here](https://github.com/chanzuckerberg/cell-census/blob/8ade9277a9235bba952b9120b01676414829cac3/tools/cell_census_builder/util.py#L61). If this change makes the addition of the trailing slash too obscure then we can keep this line.